kubernetes/cray-sysmgmt-health/values.yaml

```diff
@@ -95,20 +64,32 @@ kubectl:
     tag: 1.24.17
     pullPolicy: IfNotPresent
 
-thanos:
+logstash-exporter:
+  logstash:
+    url: http://sma-logstash-sma-logstash-headless.sma.svc.cluster.local:9600
```

Comment:
This should be
http://sma-logstash-headless.sma.svc.cluster.local:9600

Reply:
Changed.

```
# curl "http://sma-logstash-headless.sma.svc.cluster.local:9600"
{"host":"sma-logstash-1","version":"8.4.0","http_address":"0.0.0.0:9600","id":"7327a8cb-6415-4a3c-9236-f8ef8a40f51b","name":"sma-logstash-1","ephemeral_id":"ee83b2bd-f099-4c49-96ce-b09431ac97b4","status":"green","snapshot":false,"pipeline":{"workers":2,"batch_size":125,"batch_delay":50},"build_date":"2022-08-19T19:24:47Z","build_sha":"68d2c67222de5ff08adc99e491ce5cc23f0d003b","build_snapshot":false}#
```